### PR TITLE
RP, PO: Remove duplicate create-icon mixin

### PR DESCRIPTION
**Pull request by nowarkdev**
> Thanks for sending a pull request!
 > Please make sure you click the contribution guidelines link above, then fill out the blanks below

fixes #

## What this PR resolves

@roryp2  Patrick

Removes duplicate create-icon mixin. Fixes the one set in mixins to actually use the $height parameter.

## Screenshot before PR (if applicable):
screen shot 2016-03-07 at 11 04 59

## Screenshot after PR:
screen shot 2016-03-07 at 11 04 42

### CODE REVIEW

File: src/utils/_mixins.scss [hunk 1]
```diff
@@ -1,13 +1,21 @@
 $tablet: 768px;
 $desktop: 1120px;
 
+%icon {
+  display: inline-block;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center;
+}
+
```
Comment (peter-mouland) [inline src/utils/_mixins.scss]: Is this being used anywhere else apart from the create-icon mixin?
Does it make sense to remove the 'extends' and put the code directly in the mixin?

Comment (nowarkdev) [inline src/utils/_mixins.scss]: Have now moved it into the mixin

@roryp2  Patrick

**Comment by peter-mouland** [comment]
:+1:

**Comment by nowarkdev** [comment]
:+1: 
@rogerfranklin2 @yhzgirl

**Pull request merged**